artichoke-backend/src/types.rs

```diff
@@ -278,9 +293,12 @@ mod tests {
         assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(root.inner()));
         let stderror = interp.eval(b"StandardError.new").unwrap();
         assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(stderror.inner()));
-        let index = interp.eval(b"IndexError.new").unwrap();
-        assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(index.inner()));
-        let domain = interp.eval(b"Math::DomainError.new").unwrap();
-        assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(domain.inner()));
+        #[cfg(feature = "stlib-cmath")]
+        {
+            let index = interp.eval(b"IndexError.new").unwrap();
+            assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(index.inner()));
+            let domain = interp.eval(b"Math::DomainError.new").unwrap();
+            assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(domain.inner()));
+        }
```

Comment:
This index error doesn't need to be wrapped in the `cfg`. `IndexError` is a core `Exception` type.
```suggestion
        let index = interp.eval(b"IndexError.new").unwrap();
        assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(index.inner()));
        #[cfg(feature = "stlib-cmath")]
        {
            let domain = interp.eval(b"Math::DomainError.new").unwrap();
            assert_eq!(Ruby::Exception, types::ruby_from_mrb_value(domain.inner()));
        }
```